### PR TITLE
(refactor) better organize strings with i18next format

### PR DESCRIPTION
**Pull request by gasolin**
i18next has more features than chrome extension localization, one of that is `organize strings with domain`

We used `view + String` directly for all strings, but its harder to recognize due to longer string name.
i18next allow us to organize strings like

{
  'overview': {
    'amount': '....'
  },
  'wallet': {
  },
  'settings': {
  }
}

and reference the string like `overview.amount` instead of `overviewAmount` in current json file

In WIP I've refactored `Overview` page for a sneak peak,
@jagodarybacka @henryboldi if you like it, I can finish this refactory

### CODE REVIEW

**Comment by henryboldi** [comment]
Looks excellent! Likely will merge after I test this tomorrow  🙌

**Comment by gasolin** [comment]
@henryboldi please wait until I fully refactored all existing strings

**Comment by Shadowfiend** [comment]
@gasolin don't worry, we do not merge draft PRs, so as long as you're in draft mode you're good :)

**Comment by henryboldi** [comment]
Oh right, it's a draft! Was feeling trigger happy 😅

File: ui/_locales/en/messages.json [hunk 1]
```diff
@@ -1,55 +1,71 @@
...
+    "estimatedFee": "Estimated network fee",
+    "sendButton": "Send",
+    "receiveAddress": "Receive address",
+    "receiveAddressDesc": "Only send Ethereum Mainnet compatible assets to this address."
+  },
+  "swap": {
+    "title": "Swap Assets",
+    "from": "Swap from:",
+    "to": "Swap to:",
+    "errorNoSellAsset": "Attempting to approve transfer without a sell asset.",
```
Comment (jagodarybacka) [inline ui/_locales/en/messages.json:37]: maybe we can nest `error...` strings?

**Review by jagodarybacka — CHANGES_REQUESTED** [review]
Awsome refactor ✨ 

just one more place has been omitted 😄 
image

**Review by jagodarybacka — APPROVED** [review]
Works fine 🚢

**Pull request merged**